font.css

```diff
@@ -0,0 +1,46 @@
+@font-face {
+  font-family: 'montserratbold';
+  src: 
+    url('fonts/montserrat-bold.woff2') format('woff2'),
+    url('fonts/montserrat-bold.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+
+}
+
+@font-face {
+  font-family: 'montserratlight';
+  src: 
+    url('fonts/montserrat-light.woff2') format('woff2'),
+    url('fonts/montserrat-light.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+
+}
+
+@font-face {
+  font-family: 'montserratmedium';
+  src: 
+    url('fonts/montserrat-medium.woff2') format('woff2'),
+    url('fonts/montserrat-medium.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'montserratregular';
+  src: 
+   url('fonts/montserrat-regular.woff2') format('woff2'),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)